docs/Editing-Configuration-Files.md

```diff
@@ -73,7 +73,7 @@ Here is a sample of the three basic types: respectively Boolean, Number and Stri
    _Note: transmission-daemon only._
 
 #### Misc
- * **cache-size-mb:** Number (default = 4), in megabytes, to allocate for Transmission's memory cache. The cache is used to help batch disk IO together, so increasing the cache size can be used to reduce the number of disk reads and writes. The value is the total available to the Transmission instance. Setting this to 0 bypasses the cache, which may be useful if your filesystem already has a cache layer that aggregates transactions.
+ * **cache-size-mb:** Number (default = 4), in megabytes, to allocate for Transmission's memory cache. The cache is used to help batch disk IO together, so increasing the cache size can be used to reduce the number of disk reads and writes. The value is the total available to the Transmission instance. Set it to the smallest value tolerable by the random access performance of your storage medium to minimize data loss in case Transmission quit unexpectedly. Setting this to 0 bypasses the cache, which may be useful if your filesystem already has a cache layer that aggregates transactions.
```

Comment:
I know I'm not the smartest person, but what does "tolerable by the random access performance of your storage medium" mean exactly? How does one determine this value? If it's meant to scare me off from adjusting the option for the lack of proper understanding then it certainly does its job well.

Reply:
It basically means "*just* big enough so that IO does not cause system slowdowns". AFAIK people who need this option determine the value by trial and error.

> If it's meant to scare me off from adjusting the option for the lack of proper understanding then it certainly does its job well.

Maybe I went too far trying to be as exact and technically correct as possible? How would you word it?

Reply:
It's clearer now that you've explained it, but I don't have any particular suggestions. Anyway, if I'm the only one who doesn't get it we can merge and then see if someone else will come asking questions.

Reply:
Sure, I'd welcome that.